src/minidump.rs

```diff
@@ -45,8 +45,8 @@ use system_info::*;
 /// [read]: struct.Minidump.html#method.read
 /// [read_path]: struct.Minidump.html#method.read_path
 #[allow(dead_code)]
-pub struct Minidump {
-    reader : Box<Readable + 'static>,
+pub struct Minidump<T : Readable> {
+    reader : T,
```

Comment:
FYI if you wind up submitting any other PRs, feel free to fix my weird non-standard formatting here. (I'm not sure where I got the idea that I needed spaces on both sides of the colon.)